pkg/fanal/analyzer/pkg/apk/apk.go

```diff
@@ -66,6 +74,7 @@ func (a alpinePkgAnalyzer) parseApkInfo(scanner *bufio.Scanner) ([]types.Package
 				pkgs = append(pkgs, pkg)
 			}
 			pkg = types.Package{}
+			patchSystemInstalledFiles(&pkg, []string{}, opts)
```

Comment:
Is it needed? `pkg` seems to be already reset above.

Reply:
It should be removed as well